protocol/communities_messenger_test.go

```diff
@@ -3447,47 +3447,49 @@ func (s *MessengerCommunitiesSuite) TestStartCommunityRekeyLoop() {
 	s.Require().False(c.Encrypted())
 	// TODO some check that there are no keys for the community. Alt for s.Require().Zero(c.RekeyedAt().Unix())
 
-	// Update the community to use encryption and check the values
```

Comment:
were your new changes causing this test to fail?

Reply:
It didn't compile because I removed `UpdateCommunityEncryption`. I will address that in follow-up.